fmriprep/cli/parser.py

```diff
@@ -313,19 +334,32 @@ def _slice_time_ref(value, parser):
     )
     g_conf.add_argument(
         "--bold2t1w-init",
-        action="store",
-        default="register",
+        action=DeprecatedAction,
         choices=["register", "header"],
-        help='Either "register" (the default) to initialize volumes at center or "header"'
-        " to use the header information when coregistering BOLD to T1w images.",
+        help="Deprecated - use `--bold2anat-init` instead.",
     )
     g_conf.add_argument(
         "--bold2t1w-dof",
+        action=DeprecatedAction,
+        choices=[6, 9, 12],
+        type=int,
+        help="Deprecated - use `--bold2anat-dof` instead.",
+    )
+    g_conf.add_argument(
+        "--bold2anat-init",
+        choices=["auto", "t1w", "header"],
```

Comment:
Do we want to allow forcing t2w as well? It's simple enough to implement.

```suggestion
        choices=["auto", "t1w", "t2w", "header"],
```

Reply:
The way I see it, `auto` signifies "use T2w but don't fail if there isn't one", whereas a dedicated `t2w` option should fail if there isn't one available - do you think this would be useful?

Reply:
The use-case I can imagine is "I want to use t2w, but I would rather get an error than silent fallback if it's unavailable." I'm not sure how common that need would be, but I don't see a good reason to disable it, and adding it later would require another release.